src/main/kotlin/me/zeroeightsix/kami/feature/module/DiscordStatus.kt

```diff
@@ -0,0 +1,119 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.KamiMod
+import me.zeroeightsix.kami.event.TickEvent
+import net.arikia.dev.drpc.DiscordEventHandlers
+import net.arikia.dev.drpc.DiscordRPC
+import net.arikia.dev.drpc.DiscordRichPresence
+import net.arikia.dev.drpc.DiscordUser
+import net.minecraft.util.Util.OperatingSystem
+import net.minecraft.util.Util.getOperatingSystem
+
+
+@Module.Info(name = "DiscordRPC", category = Module.Category.MISC, description = "Discord Rich presence")
+class DiscordStatus : Module() {
+    @Setting(name = "Version")
+    private var version = true
+
+    @Setting(name = "Username")
+    private var username = true
+
+    @Setting(name = "Server")
+    private var server = true
+
+    @Setting(name = "Flair")
+    private var message = true
+
+    var lastUpdate: Long = 0L
+    var rpcEnabled = false
+
+    val messages = arrayOf(
```

Comment:
these should be user configurable

Reply:
No, the reason its what it is is discord's reccomended limit to stop api spam.

Reply:
what?????